pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/nullvalue/NullValueIndexTypeTest.java

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
in addition, let's add some tests to the NullHandling.json test file to validate the handling of null values are being considered